README.md

```diff
@@ -39,6 +39,7 @@ The examples folder contains example solutions across a variety of Google Cloud
 * [Dataflow Template Pipelines](https://github.com/GoogleCloudPlatform/DataflowTemplates) - Pre-implemented Dataflow template pipelines for solving common data tasks on Google Cloud Platform.
 * [Dataproc GCS Connector](examples/dataproc-gcs-connector) - Install and test unreleased features on the GCS Connector for Dataproc.
 * [Dataproc Persistent History Server for Ephemeral Clusters](examples/dataproc-persistent-history-server) - Example of writing logs from an ephemeral cluster to GCS and using a separate single node cluster to look at Spark and YARN History UIs.
+* [DFDL Processing with Firestore and Pubsub](examples/dfdl-firestore-pubsub-example) - Example to process a binary using DFDL definition using Daffodil libraries. The DFDL definition is stored in firestore, the request to process is done through a pubsub sub and the output is published is a JSON format in Pubsub topic.
```

Comment:
I would include an expansion of the acronym DFDL. Apologies for my ignorance, maybe it is a well known acronym, but I am unfamiliar with it.

Reply:
Done